packages/app/src/components/time-series-chart/components/tooltip/tooltip-series-list.tsx

```diff
@@ -169,7 +169,11 @@ function TooltipListItem({
             <VisuallyHidden>
               <InlineText mr={2}>{label}:</InlineText>
             </VisuallyHidden>
-            <TooltipEntryValue>{children}</TooltipEntryValue>
+            <TooltipEntryValue
+              displayTooltipValueOnly={displayTooltipValueOnly}
+            >
```

Comment:
`isCentered` makes more sense as api of the TooltipEntryValue component imho;

```suggestion
            <TooltipEntryValue isCentered={displayTooltipValueOnly}>
```